packages/keys/src/key-pair.ts

```diff
@@ -9,3 +11,15 @@ export async function generateKeyPair(): Promise<CryptoKeyPair> {
     );
     return keyPair as CryptoKeyPair;
 }
+
+export async function createKeyPairFromBytes(bytes: Uint8Array, extractable?: boolean): Promise<CryptoKeyPair> {
+    if (bytes.length !== 64) {
+        // TODO: Coded error.
+        throw new Error('KeyPair bytes must be of length 64');
```

Comment:
Coded errors are gtg if you want to write your first one! Just land the whole thing in the same PR. Add to `./codes.ts`, `./messages.ts` and `./context.ts` if you want to add some.

Reply:
Done!